idrstream/DP_idr.py

```diff
@@ -340,7 +349,61 @@ def clear_batch(self):
             shutil.rmtree(path)
 
         self.logger.info("Temporary batch files cleared")
-    
+
+    def add_batch_object_outlines(
+        self, batch_single_cell_df: pd.DataFrame, object_metadata_channel="Metadata_DNA"
+    ) -> pd.DataFrame:
+        """
+        add object outlines to single cell data for current batch
+        necessary because we compile original single cell df with pycytominer
+        but pycytominer cannot append additional metadata (like object outlines)
+        Parameters
+        ----------
+        batch_single_cell_df : pd.DataFrame
+            original batch single cell df
+        object_metadata_channel : str
+            DeepProfiler channel name (from index.csv) to get object outlines from
+        Returns
+        -------
```

Comment:
Minor suggestion: keep the spacing consistent for your method/function docstrings.
```suggestion
        but pycytominer cannot append additional metadata (like object outlines)
        
        Parameters
        ----------
        batch_single_cell_df : pd.DataFrame
            original batch single cell df
        object_metadata_channel : str
            DeepProfiler channel name (from index.csv) to get object outlines from
        
        Returns
        -------
```